components/fan/hbridge.rst

```diff
@@ -23,6 +23,14 @@ The `'hbridge`' fan platform allows you to use a compatible `h-bridge` (L298N, D
 .. code-block:: yaml
 
     # Example configuration entry
+    output:
+      - platform: esp8266_pwm
+        id: motor_forward_pin
+        pin: D1
+      - platform: esp8266_pwm
+        id: motor_reverse_pin
+        pin: D2
+    
```

Comment:
Would prefer not to mention the specific platform and use GPIO pins not board aliases

```suggestion
    output:
      - platform: ...
        id: motor_forward_pin
        pin: GPIO5
      - platform: ...
        id: motor_reverse_pin
        pin: GPIO4
    
```